### PR TITLE
Add 0.15 release notes for UI scrolling

### PR DESCRIPTION
**Pull request by Piefayth**
Luckily there's not too much API surface to discuss here. My first release note, so let me know what I need to change!

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
This is solid and appropriately brief. This might be a nice spot for a polished screenshot, but that can be added later.

File: release-content/0.15/release-notes/_release-notes.toml [hunk 1]
```diff
@@ -429,3 +429,10 @@ authors = ["@cart"]
 contributors = ["@alice-i-cecile", "@killercup", "@BD103"]
 prs = [14791]
 file_name = "14791_Required_Components.md"
+
+[[release_notes]]
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/_release-notes.toml:433]: Double check that we don't have a duplicate in here: the stub should already have been generated.

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Actually, one quick request: I think you duplicated the release note stub?

**Comment by Piefayth** [comment]
Whoops. Fixed.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**